src/OpenApi3/WhitelistedSchema.php

```diff
@@ -41,10 +42,31 @@ public function addOperationRelations(OperationGuess $operationGuess, Registry $
             return;
         }
 
+        /** @var RequestBody|null $requestBody */
+        $requestBody = $operationGuess->getOperation()->getRequestBody();
+        if (null !== $requestBody) {
+            if (null !== $requestBody->getContent() && is_iterable($requestBody->getContent())) {
+                /** @var MediaType $content */
+                foreach ($requestBody->getContent() as $contentType => $content) {
+                    if ('application/json' === $contentType) {
```

Comment:
Could you add support for `application/x-www-form-urlencoded` aswell ? Following code would be the same here I thin, you just have to let this case pass this if condition :wink:

Reply:
Done ;)